pithos/pithos.py

```diff
@@ -474,6 +474,10 @@ def init_actions(self, app):
         app.add_accelerator('<Primary>d', 'win.bookmark', None)
         action.connect('activate', self.bookmark_song)
 
+        action = Gio.SimpleAction.new('station-popover', None)
+        self.add_action(action)
+        app.add_accelerator('<Primary>r', 'win.station-popover', None)
```

Comment:
The action name can be improved to be descriptive, maybe `win.toggle-station-popover`